python/ray/train/v2/api/config.py

```diff
@@ -13,10 +15,15 @@
     TRAINER_RESOURCES_DEPRECATION_MESSAGE,
 )
 from ray.train.v2._internal.util import date_str
+from ray.util.annotations import PublicAPI
 
 if TYPE_CHECKING:
     from ray.train import UserCallback
 
+import pyarrow.fs
```

Comment:
can we put external library imports between python standard imports and ray imports?

```python
from typing import TYPE_CHECKING, List, Optional, Union

import pyarrow.fs

from ray.air.config import (
```

I thought the linter isort does this already 🤔